.phpcs.xml

```diff
@@ -45,6 +45,9 @@
 		<exclude name="WordPress.Files.FileName.InvalidClassFileName">
 			<exclude-pattern>*/*</exclude-pattern>
 		</exclude>
+		<exclude name="WordPressVIPMinimum.Performance.FetchingRemoteData.FileGetContentsUnknown">
+			<exclude-pattern>*/*</exclude-pattern>
+		</exclude>
```

Comment:
Assuming this is thrown by something in the test, can we exclude there or only for tests, rather than globally?